pom.xml

```diff
@@ -3,8 +3,7 @@
     <parent>
         <artifactId>eblocker-top</artifactId>
         <groupId>org.eblocker</groupId>
-        <version>2.4.12</version>
-        <relativePath>../eblocker-top</relativePath>
```

Comment:
The relative reference is omitted, as the package is now obtained directly from the release of github.